src/btree/bt_curprev.c

```diff
@@ -220,20 +219,19 @@ __cursor_fix_append_prev(WT_CURSOR_BTREE *cbt, bool newpage, bool restart)
 static inline int
 __cursor_fix_prev(WT_CURSOR_BTREE *cbt, bool newpage, bool restart)
 {
-    WT_BTREE *btree;
     WT_PAGE *page;
     WT_SESSION_IMPL *session;
 
     session = CUR2S(cbt);
     page = cbt->ref->page;
-    btree = S2BT(session);
 
     /* If restarting after a prepare conflict, jump to the right spot. */
     if (restart)
         goto restart_read;
 
     /* Initialize for each new page. */
     if (newpage) {
+        cbt->slot = UINT32_MAX;
```

Comment:
It matches the VLCS code, and in both cases it's because there's a way out of the function before it gets set again. I should probably copy the comment (which is present in both the VLCS and row code). (Done)

Reviewing cbt.slot everywhere also seems like a good idea -- IIRC we talked about it at some point before. If nothing else it would be nice to change the UINT32_MAX uses to WT_SLOT_INVALID or something like that.

Reply:
Agreed about doing a `cbt.slot` review -- can you please add it to your list, and it's a separate ticket, of course.

Reply:
Added it yesterday :-)